File: Swift/L4/Lista_04_EX04/UsuarioRoot.swift
import Foundation
 
 public class UsuarioRoot : Usuario{
 
	override func descricao(){
		print("O usuario Root tem todas as permissões")
	}
	
	func excluirTodos(){
		print("O usuario Root pode excluir qualquer usuario")
	}
	
	func despedidaRoot( ) {
		print("Até logo \(login)!")
	}
	
}

File: Java/Lista 4/L4_Ex1.4/src/l4_ex1/pkg4/UsuarioRoot.java

package l4_ex1.pkg4;

public class UsuarioRoot extends Usuario {
    
    public UsuarioRoot(String login) {
        super(login);
    }
    @Override
    public void descricao(){
        System.out.println("O usuario Root tem todas as permissões");
    }
    public void excluirTodos(){
        System.out.println("O usuario Root pode excluir qualquer usuario");
    }
    public void despedidaRoot(){
        System.out.println("Até logo " + this.getLogin() + "!");
    }
    
}